rpcs3/emucore.vcxproj.filters

```diff
@@ -1264,6 +1264,7 @@
     <ClCompile Include="Emu\Io\MouseHandler.cpp">
       <Filter>Emu\Io</Filter>
     </ClCompile>
+    <ClCompile Include="rpcs3qt\rpcn_countries.cpp" />

```

Comment:
This makes no sense. you're putting the file into rpcs3qt but then you compile them as part of emucore.

Reply:
Ok, will adjust accordingly.